client/src/components/mapSidePanel.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable require-jsdoc */
+// Todo VS: what is a better way around this jsdoc disabling?
```

Comment:
Would like to review how to remove this `es-lint-disable`. Sat down with Nat on this earlier, but still scratching my head...

Reply:
looks like [tsdoc](https://github.com/microsoft/tsdoc) is a pretty good option here - maybe you can make up a ticket to convert to using this and leave a TODO?